install_config/aggregate_logging.adoc

```diff
@@ -849,19 +849,103 @@ and the xref:#deploying-the-efk-stack[{product-title} logging installer] creates
+
 [WARNING]
 ====
 Using NFS storage as a volume or a persistent volume, or using NAS such as
 Gluster, is not supported for Elasticsearch storage, as Lucene relies on file
 system behavior that NFS does not supply. Data corruption and other problems can
-occur. If you must use NFS storage, allocate a large file on a
-volume to serve as a storage device and mount it locally on one host.
-For example, if your NFS storage volume is mounted at *_/nfs/storage_*:
+occur. 
+====
+
+[[aggregated-logging-nfs-persistent]]
+*Using NFS as persistent volume*
+
+You can deploy NFS as an automatically provisioned persistent volume or using a predefined NFS volume.
+
+For more information, see xref:../install_config/storage_examples/shared_storage.adoc#install-config-storage-examples-shared-storage[leverage shared storage] to leverage shared storage for use by two separate containers.
+
+To use NFS as persistent volume where NFS is automatically provisioned:
+
+. Edit the Ansible inventory file to create an NFS auto-provision storage class and set the inventory variable.
++
+----
+openshift_logging_es_pvc_storage_class_name=$nfsclass
+openshift_logging_es_pvc_dynamic=true
+----
+
+. Deploy the NFS volume using the logging playbook.
++
+----
+ansible-playbook /usr/share/ansible/openshift-ansible/playbooks/openshift-logging/config.yml
+----
+
+. Use the following steps to create a PVC:
+
+.. Edit the Ansible inventory file to set the PVC size:
++
+----
+openshift_logging_es_pvc_size=50Gi
+----
++
+[NOTE]
+====
+The logging playbook selects a volume base on size and might use an unexpected volume if any other persistent volume has same size.
+====
 
+.. Deploy logging using NFS volume using playbook.
++
+----
+ansible-playbook /usr/share/ansible/openshift-ansible/playbooks/openshift-logging/config.yml
 ----
-$ truncate -s 1T /nfs/storage/elasticsearch-1
-$ mkfs.xfs /nfs/storage/elasticsearch-1
-$ mount -o loop /nfs/storage/elasticsearch-1 /usr/local/es-storage
++
+The installer playbook create the NFS volume based on the `openshift_logging_storage` variables.
++
+[NOTE]
+====
+The logging playbook selects a volume base on size and might use an unexpected volume if any other persistent volume has same size.
+====
+
+To deploy logging alongside the {product-title} cluster using the NFS volume: 
+
+. Edit the Ansible inventory file to set the PVC size:
++
+----
+openshift_logging_storage_kind=nfs
+openshift_enable_unsupported_configurations=true
+openshift_logging_storage_access_modes=["ReadWriteOnce"]
+openshift_logging_storage_nfs_directory=/srv/nfs
+openshift_logging_storage_nfs_options=*(rw,root_squash)
+openshift_logging_storage_volume_name=logging
+openshift_logging_storage_volume_size=100Gi
+openshift_logging_storage_labels={:storage=>"logging"}
+openshift_logging_install_logging=true
+----
+
+. Deploy logging using NFS volume using playbook.
++
+----
+ansible-playbook /usr/share/ansible/openshift-ansible/playbooks/openshift-logging/config.yml
```

Comment:
openshift_logging_storage_*  variables don't affect the logging playbook. It can only affect the installer playbook. You must set these variables, and deploy/redeploy cluster using playbook  /usr/share/ansible/openshift-ansible/playbooks/deploy_cluster.yml